extensions/Vendor/EXT_texture_transform/schema/EXT_texture_transform.textureInfo.schema.json

```diff
@@ -26,7 +28,8 @@
         "texCoord": {
             "type": "integer",
             "description": "Overrides the textureInfo texCoord value if this extension is supported.",
-            "default": 0
+            "default": 0,
```

Comment:
I suggest to remove default value here, because it should default to texture's texCoord which isn't necessarily zero.